.ci/kitchen-centos7-py3-m2crypto

```diff
@@ -1,16 +0,0 @@
-@Library('salt@master-1.7') _
```

Comment:
Why are we removing m2crypto tests as well? A user can still use the m2crypto library or pycryptodomex.

Reply:
From what I understood, the goal, with or without salt-bin, but specially with salt-bin, is to focus on **one** supported crypto library, no?

Reply:
If it was a goal to only support only one library I just wasn't aware. But users can still use m2crypto if they are using pip or if they pip install it into the binary. And we try to import it first (https://github.com/saltstack/salt/blob/master/salt/crypt.py#L57).

Reply:
Hmm, maybe it was just wishful thinking.
Let's bring this up in standup today.

Reply:
We still want to support M2Crypto for the time being as far as I recall.

Reply:
PR updated